documentation/src/main/asciidoc/user_guide/chapter-78-Scripting.adoc

```diff
@@ -41,6 +41,13 @@ The following metadata property keys are available
 * collator: this property is only valid for +mapper+ scripts, and specifies the name of another script that will be used for the collation phase of a map/reduce job. This property will be removed in Infinispan 9.
 * combiner: this property is only valid for +mapper+ scripts, and specifies the name of another script that will be used for the combine phase of a map/reduce job. This property will be removed in Infinispan 9.
 * parameters: an array of valid parameter names for this script. Invocations which specify parameter names not included in this list will cause an exception.
+* datatype: optional property providing information, in the form of
```

Comment:
Yeah, that would be a good idea, but @rigazilla remember one thing: you still need the JBasicMarshaller to be able to store the script remotely using the Hot Rod protocol at least with the default marshaller on the server. If the script can be stored using CLI operations, then we could get rid of JBasicMarshaller altogether but dunno if that's possible...